apps/console/src/features/server-configurations/pages/connector-listing-page.tsx

```diff
@@ -0,0 +1,433 @@
+import { GridLayout, PageLayout } from "@wso2is/react-components";
+import { AxiosError } from "axios";
+import React, { FunctionComponent, MutableRefObject, ReactElement, useEffect, useRef, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useDispatch, useSelector } from "react-redux";
+import { Dispatch } from "redux";
+import { Divider, Grid, Placeholder, Ref } from "semantic-ui-react";
+import UsernameValidationIcon from "../../../extensions/assets/images/icons/username-validation-icon.svg";
+import { serverConfigurationConfig } from "../../../extensions/configs/server-configuration";
+import { AppConstants, AppState, FeatureConfigInterface, history, store } from "../../core";
+import { PrivateKeyJWTConfig } from "../../private-key-jwt/pages/private-key-jwt-config";
+import { ValidationConfigPage } from "../../validation/pages/validation-config";
+import { getConnectorCategories, getConnectorCategory } from "../api";
+import { ServerConfigurationsConstants } from "../constants";
+import {
+    GovernanceConnectorCategoryInterface,
+    GovernanceConnectorInterface
+} from "../models";
+import { EditConnector } from "../settings/edit-connector";
+import { SettingsSection } from "../settings/settings-section";
+
+/**
+ * Props for the Server Configurations page.
+ */
+type ConnectorListingPageInterface = TestableComponentInterface;
+
+/**
+ * `GovernanceConnectorInterface` type with `ref` attr.
+ */
+type GovernanceConnectorWithRef = GovernanceConnectorInterface & ReferableComponentInterface<HTMLDivElement>;
+
+/**
+ * Governance connector listing page.
+ *
+ * @param props - Props injected to the component.
+ * @returns Governance connector listing page component.
+ */
+export const ConnectorListingPage: FunctionComponent<ConnectorListingPageInterface> = (
+    props: ConnectorListingPageInterface
+): ReactElement => {
+    const { [ "data-testid" ]: testId } = props;
+
+    const dispatch: Dispatch = useDispatch();
+    const pageContextRef: MutableRefObject<any> = useRef(null);
+
+    const { t } = useTranslation();
+
+    const featureConfig: FeatureConfigInterface = useSelector((state: AppState) => state.config.ui.features);
+    const allowedScopes: string = useSelector((state: AppState) => state?.auth?.allowedScopes);
+
+    const [ connectorCategories, setConnectorCategories ] = useState<GovernanceConnectorCategoryInterface[]>([]);
+    const [ connectors, setConnectors ] = useState<GovernanceConnectorWithRef[]>([]);
+    const [ selectedConnector, setSelectorConnector ] = useState<GovernanceConnectorWithRef>(null);
+
+    // const [ categoryId, setCategoryId ] = useState<string>("");
+    const [ isConnectorCategoryLoading, setConnectorCategoryLoading ] = useState<boolean>(true);
+
+    useEffect(() => {
+
+        // If Governance Connector read permission is not available, prevent from trying to load the connectors.
+        if (!hasRequiredScopes(featureConfig?.governanceConnectors,
```

Comment:
@DonOmalVindula Is there a change to this with the recent improvements to access control.

Reply:
No... the improvement was done to the UI component of the access-control module. This is not affected